python/lsst/dax/apdb/apdbCassandra.py

```diff
@@ -288,6 +310,45 @@ def _make_auth_provider(self, config: ApdbCassandraConfig) -> AuthProvider | Non
 
         return None
 
+    def _versionCheck(self, metadata: ApdbMetadataCassandra) -> None:
+        """Check schema version compatibility."""
+
+        def _get_version(key: str, default: VersionTuple) -> VersionTuple:
+            """Retrieve version number from given metadata key."""
+            if metadata.table_exists():
+                version_str = metadata.get(key)
+                if version_str is None:
+                    # Should not happen with existing metadata table.
+                    raise RuntimeError(f"Version key {key!r} does not exist in metadata table.")
+                return VersionTuple.fromString(version_str)
+            return default
+
+        initial_version = VersionTuple(0, 1, 0)
```

Comment:
I was confused why this wasn't `VERSION`, so a comment pointing out that this is used to specify versions in old schemas when the metadata is not present would be helpful.

Reply:
👍 I'll add a comment.